assets/wizards/popups/components/segmentation-preview/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Segmentation Preview component.
+ * Extension of WebPreview with support for "view-as-segment" functionality.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies.
+ */
+import { WebPreview } from '../../../../components/src';
+
+const SegmentationPreview = props => {
+	const postPreviewLink = window?.newspack_popups_wizard_data?.preview_post;
```

Comment:
This is the first time I've seen [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) in action. Always something new to learn!

Reply:
I cannot claim credit :-), that's a lift from @adekbadek's [original Preview work](https://github.com/Automattic/newspack-plugin/pull/741/files#diff-b103111ab76a5a84285789bd2ae2cc0a5d4f29cdb5278802e195115d58cee273R30-R31)...